vanilla/scss/styles.scss

```diff
@@ -70,3 +70,45 @@ li > p {
 .admonition-security-notice {
   padding-left: 1rem;
 }
+
+// Toc custom styles
```

Comment:
This contains styles from the other unmerged PR.

Reply:
(I'll merge the other one first)